web/yo/app/scripts/services/databaseconnector.js

```diff
@@ -43,6 +45,28 @@ angular.module('oncokbApp')
             var testing = OncoKB.config.testing || false;
             var inProduction = OncoKB.config.production || false;
 
+
+            function getAllDrugs() {
+                var deferred = $q.defer();
+                Drugs.getAllDrugs()
+                    .then(function(data) {
+                        deferred.resolve(data.data);
+                    }, function(error) {//send email?
+
+                    });
+                return deferred.promise;
+            }
+
+            function searchDrugs(keyword) {
+                var deferred = $q.defer();
+                Drugs.searchDrugs(keyword)
+                    .then(function(data) {
+                        deferred.resolve(data.data);
+                    }, function(error) {//send email?
```

Comment:
Probably in the dropdown, we say the feature is not available.

Reply:
There are two actions when an error occurs. 
First, to show 'This page is unavailable now' or 'Sorry, suggested drugs are unavailable now.'.
Second, use sendEmail service to send emails.